prov/efa/src/rdm/efa_rdm_ep_utils.c

```diff
@@ -673,3 +673,16 @@ void efa_rdm_ep_post_handshake_or_queue(struct efa_rdm_ep *ep, struct efa_rdm_pe
 
 	peer->flags |= EFA_RDM_PEER_HANDSHAKE_SENT;
 }
+
+size_t efa_rdm_ep_get_memory_alignment(struct efa_rdm_ep *ep, enum fi_hmem_iface iface)
+{
+	size_t memory_alignment = 8;
```

Comment:
Can we make the 8 also a macro `EFA_RDM_DEFAULT_MEMORY_ALIGNMENT` similar to the other ones?

Reply:
Done.